test/e2e/test/e2e.test.ts

```diff
@@ -131,6 +131,30 @@ describe('End to end tests for common events', () => {
     expect(sentryEvent.eventID).toMatch(eventId);
   });
 
+  test('captureErrorReplay', async () => {
+    const element = await getElement('captureException');
+    await element.click();
+
+    const eventId = await waitForEventId();
+    const sentryEvent = await fetchEvent(eventId);
+    expect(sentryEvent.eventID).toMatch(eventId);
+
+    expect(sentryEvent.contexts).toBeDefined();
+    const replay = sentryEvent.contexts!['replay'] as any;
+    expect(replay).toBeDefined();
+    expect(replay.replay_id.length).toBe(32);
+
+    const replayInfo = await fetchReplay(replay.replay_id);
```

Comment:
I thought possibly the Android replays are uploaded due to a short time limit, but if I understand the code correctly, this fetch will wait up to 10 minutes, checking the endpoint every 1s. 

Is that right? @vaind

Reply:
yeah, nothing to do with timeouts. Replay capture doesn't work at all on android emulator in GHA at the moment

Reply:
Also no screenshots are captured by appium, if you look into an artifact for any of the Android jobs:

<img width="703" alt="image" src="https://github.com/user-attachments/assets/0d8f4f07-a2d1-4d57-bf9c-f5421a2e43f5">


From what I can tell it's an old/recurring issue of the emulator on GHA, see https://github.com/actions/runner-images/issues/6814